dallinger/frontend/static/scripts/dallinger2.js

```diff
@@ -427,6 +427,38 @@ var dallinger = (function () {
     return deferred;
   };
 
+  /**
+   * Load an existing `Participant` into the dlgr.identity by making a ``POST``
+   * request to the experiment `/participant` route with an ``assignment_id``.
+   *
+   * @returns {jQuery.Deferred} See :ref:`deferreds-label`
+   */
+  dlgr.loadParticipant = function(assignment_id) {
+    var deferred = $.Deferred(),
+        url = '/participant';
+
+    if (dlgr.identity.participantId !== undefined && dlgr.identity.participantId !== 'undefined') {
```

Comment:
Were you seeing a value of 'undefined' (as a string) assigned in some cases? I think I went on a campaign to prevent that from happening at some point, but maybe I never did, or maybe I did, but failed.

Reply:
I just copied this logic from the function above, assuming that whatever might have caused such a situation for `createParticipant` could cause the same issue here.